### PR TITLE
Add Joseph Musembi to the contributors page

### PR DESCRIPTION
**Pull request by musemby**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1669,13 +1674,3 @@ Noida
 - Place: London, England
 - Bio: Management Consultant
 - Github: [widgyrogers] (https://github.com/widgyrogers)
-
-#### Name: [Ipaye Alameen](https://github.com/ipaye)
```
Comment (StevenCopeland) [inline CONTRIBUTORS.md]: Why are you deleting this name?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -1669,13 +1674,3 @@ Noida
...
-- Place: Lagos, Nigeria
-- Bio: Coumpter Engeering Undergrad | Front-end Developer | Javascript enthusiast
-- Github: [ipaye] (https://github.com/ipaye)
-
-#### Name: [Kutsoragi](https://github.com/Kutsoragi)
```
Comment (StevenCopeland) [inline CONTRIBUTORS.md]: And why are you deleting this name?

**Comment by musemby** [comment]
@StevenCopeland my bad, I had not merged in the latest upsteam master. All good now

**Comment by Roshanjossey** [comment]
Hi  @musemby, I'm working on a similar project to help beginners to get started with contributing to open source projects.

Do check it out https://github.com/Roshanjossey/first-contributions

**Pull request merged**